packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.scss

```diff
@@ -0,0 +1,5 @@
+ovh-manager-pci-project-notebook-general-information {
+  .oui-popover {
+    word-break: break-word;
```

Comment:
**question:** Can't we avoid using custom style here?

Reply:
I don't think so, we use the `popover` directive inside the `oui-tile-description`, which has the following css:
```
.oui-tile__description {
    word-break: break-all;
}
```

Reply:
We can use `term-popover` attribute.
see: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-tile-webcomponent--with-definition-list

Reply:
If I understand correctly, `term-popover` would put a `popoever` on the `title` of the `oui-tile-description`.
I'm not sure that would work here, as we need 2 `popover`, on the content of the description:

![image](https://user-images.githubusercontent.com/24541481/158973388-4e6db313-8404-4c11-8923-31462d58415d.png)

Reply:
Issue opened here: ovh/ovh-ui-kit#774

Reply:
Custom css removed, now that the issue is fixed in the ovh-ui-kit: https://github.com/ovh/manager/pull/6676/commits/56b5bc9169ef38e5bf6d3d3e0c3c2665f6da2934 :smiley: